bindata/network/multus/multus.yaml

```diff
@@ -116,29 +116,109 @@ spec:
       tolerations:
       - operator: Exists
       serviceAccountName: multus
-      initContainers:
-      - name: multus-binary-copy
+      containers:
+      - name: kube-multus
         image: {{.MultusImage}}
-        command: ["/entrypoint/cnibincopy.sh"]
+        command: [ "/bin/bash", "-c", "--" ]
+        args:
+          - >
+            /entrypoint/cnibincopy.sh;
+            /entrypoint.sh
```

Comment:
don't you want to `exec /entrypoint.sh ...` this, so you don't have to worry about Bash sitting in the middle and possibly complicating signal handling?

Reply:
So, this is why I had the init container originally, was that it simplified this process, that I want the installation of the binary, and the arguments for the configuration of Multus to be separate.... any further ideas on this one on how I might do it without the entrypoint?

Reply:
I would generally do:

```
command:
- /bin/bash
- -c
-|
  set -e
  <env> /entrypoint/cnibincopy.sh
  exec /entrypoint.sh $@
args:
- ...
```

That preserves the existing behavior of both.

Reply:
gah, I was being dense. I added the bash command `exec` for running entrypoint.sh as suggested, thanks :+1: